kubejs/assets/ftbquests/lang/pl_pl.json

```diff
@@ -0,0 +1,130 @@
+    "chapter.1.quest.19.subtitle": "Laky docenia WSZYSTKICH",
+    "chapter.1.quest.19.task.1.title": "Rzeczy do zapamiętania!",
+    "chapter.1.quest.19.title": "Specjalne podziękowania",
+    "chapter.1.quest.2.description.1": "Konstruktor części jest miejscem, w którym bierzesz wzór, bierzesz materiały bazowe i tworzysz indywidualne komponenty narzędzia.",
+    "chapter.1.quest.2.subtitle": "Zbuduj części swojego narzędzia z tym.",
+    "chapter.1.quest.20.title": "Zbierz! 64 bloki melona",
+    "chapter.1.quest.21.title": "Zbierz!! 64 kaktusy",
+    "chapter.1.quest.22.title": "Zbierz!! 64 dynie",
+    "chapter.1.quest.23.title": "Zbierz!! 64 skrzynki marchewkowe",
+    "chapter.1.quest.24.title": "Zbierz!! 64 skrzynki ziemniaczane",
+    "chapter.1.quest.25.title": "Zbierz!! 64 skrzynki kukurydziane",
+    "chapter.1.quest.26.title": "Zbierz!! 64 worki ryżowe",
+    "chapter.1.quest.27.title": "Zbierz!! 64 skrzynki cebulowe",
+    "chapter.1.quest.28.title": "Zbierz!! 64 skrzynki pomidorowe",
+    "chapter.1.quest.29.title": "Zbierz!! 64 skrzynki kapuśniaczane",
+    "chapter.1.quest.3.description.1": "Stacja Tinker jest miejscem, w którym montujesz swoje narzędzia Tinkers', które będą potrzebne do uzyskania kamienia, miedzi i żelaza. Narzędzia drewniane i kamienne są pierwszym celem, ponieważ receptury vanilla tych narzędzi już nie istnieją.\n\n\nMożesz również naprawiać narzędzia Tinkers' w tej stacji, umieszczając swoje narzędzie w prawej szczelinie i materiały w dowolnej innej szczelinie. To jest super wygodne! Możesz również zainwestować w zestawy naprawcze, aby naprawiać narzędzia Tinkers' bezpośrednio z twojego ekwipunku!",
+    "chapter.1.quest.3.subtitle": "Zmontuj swoje narzędzia z tym.",
+    "chapter.1.quest.30.description.1": "(Och, to było warte tego.)",
+    "chapter.1.quest.30.subtitle": "Czy to było warte?",
+    "chapter.1.quest.30.title": "Bardzo ładny przypraw",
+    "chapter.1.quest.31.description.1": "Te plecaki zajmują miejsce, ale zawierają ogromną ilość miejsca i wbudowany crafting. Upewnij się, że przeczytasz opis, aby wiedzieć, jak je wyposażyć.\n\n\nGdy tylko zbudujesz coś w plecaku, przedmiot nie pojawi się. Gdy tylko zostanie zbudowany, przedmiot pojawi się. Mamy nadzieję, że obraz kota zamiast zbudowanego przedmiotu ułatwi twoją ból.\n\n\nOstrzeżenie: Użycie obiektu Piggybackpack Tinkers', gdy plecak jest na twoich plecach, zniszczy plecak i wszystko, co zawiera.",
+    "chapter.1.quest.31.subtitle": "Portowe magazyny? Świetnie...",
```

Comment:
I think there original word was Portable - "Przenośne magazyny" is a better translation.